### DISCUSSION TITLE
Feauture Request: Dot and line plots

### DISCUSSION THREAD

**Discussion by Auerilas** [mwaskom/seaborn#3125] [Q&A]
Not sure what they are officially called, but I like this style of plots showing the individual data with lines marking the centroid (mean/median) and the error (SD, SEM, etc).

[anpp_grass.pdf](https://github.com/mwaskom/seaborn/files/9933019/anpp_grass.pdf)

Is this a feature that could be added? The strip plot allows the dots, but the part for the lines doesn't exist so far as I could tell.

**Comment by jcmkk3** [comment]
I've seen those (not necessarily with the jittered dots) called TIE fighter plots.

This is not exactly what you're looking for, but maybe with some of these techniques you could get close. It is probably worth messing around with [`Est`](https://seaborn.pydata.org//generated/seaborn.objects.Est.html) object.

```python
import seaborn.objects as so
from seaborn import load_dataset

penguins = load_dataset("penguins")

(
    so.Plot(penguins, x="species", y="body_mass_g", color="sex")
    .add(so.Dot(alpha=0.5), so.Jitter(), so.Dodge())
    .add(so.Range(), so.Perc([20, 80]), so.Dodge())
    .add(so.Dash(), so.Perc([20, 50, 80]), so.Dodge())
    .label(x=None, y="Body Mass (g)", color=None)
)
```

![image](https://user-images.githubusercontent.com/45919695/199850171-56d04d9d-83d7-4efa-9fb6-70ad27980737.png)

**Comment by Auerilas** [comment]
This looks great! I haven't played with the new syntax yet, but the graph
you produced here was considerably easier than my forcing it in matplotlib.

**Comment by mwaskom** [comment]
And in the classic interface, it's not precisely what you are asking for, but this is basically a `pointplot`?

```python
spec = dict(data=penguins, x="species", y="body_mass_g", hue="sex", dodge=.4)
sns.catplot(**spec, alpha=.25, legend=False)
sns.pointplot(**spec, capsize=.3, errorbar="sd", join=False)
```
![image](https://user-images.githubusercontent.com/315810/199857581-2a200f5e-d1f0-49b8-b8d6-b6fe61176802.png)
